tests/testthat/testImportCNA.GISTIC2.R

```diff
@@ -0,0 +1,45 @@
+#
+# test importCNA.GISTIC2.R
+#
+library(testthat)
+context( "import rCNA data from GISTIC2 files")
+#
+#
+
```

Comment:
First set of tests: invalid input is rejected.

`test_that( "importCNA.GISTIC2 rejects invalid input ...")
# missing filename
# invalid filename
# file does not exist
# missing target directory
# invalid target directory
# directory does not exist
`